flatgfa/src/cli/cmds.rs

```diff
@@ -299,3 +301,50 @@ pub fn gaf_lookup(gfa: &flatgfa::FlatGFA, args: GAFLookup) {
         }
     }
 }
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-import")]
+pub struct SeqImport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_import(args: SeqImport) {
+    let vec = packedseq::import(&args.filename);
+    let _ = std::fs::remove_file(&args.filename);
+    let store = PackedSeqStore::create(vec);
+    let view = store.as_ref();
+    print!("{}", view);
+}
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-export")]
+pub struct SeqExport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_export(args: SeqExport) {
+    let mut input = String::new();
+    std::io::stdin()
+        .read_to_string(&mut input)
+        .expect("Stdin read failure");
+
+    let vec: Vec<packedseq::Nucleotide> = input
+        .chars()
+        .filter(|c| !c.is_whitespace())
+        .map(|c| match c {
+            'A' => packedseq::Nucleotide::A,
+            'C' => packedseq::Nucleotide::C,
+            'T' => packedseq::Nucleotide::T,
+            'G' => packedseq::Nucleotide::G,
+            _ => panic!("Invalid character in fle: {}", c),
+        })
+        .collect();
+
+    let store = packedseq::PackedSeqStore::create(vec);
```

Comment:
I suggest that we probably want a method on `PackedSeqStore` that does this conversion, from a string or `&[u8]` to compressed data. As an added bonus, it could avoid allocating a `Vec<Nucleotide>` at all. Consider something like this:

I suggest that we probably want a method on `PackedSeqStore` that does this conversion, from a string or `&[u8]` to compressed data. As an added bonus, it could avoid allocating a `Vec<Nucleotide>` at all. Consider something very roughly like this:

```rs
fn append_text(&mut self, text_data: &[u8]) {
    for c in text_data {
        self.push(c.into());
    }
}

fn from_text(text_data: &[u8]) -> Self {
    let mut store = Self::new();
    store.append_text(text_data);
    store
}
```